Bottom: 4f8a12066e4f5d857c4bbe6e3c51faefbba051a0
Top:    028e02290441e2b23cd317d31c0e963cf55439a0
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:24:22 +0000

Remove debug from outline generation


---

diff --git a/src/polygon.c b/src/polygon.c
index 65edb50..2bca1f3 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -108,7 +108,7 @@ dicer output is used for HIDs which cannot render things with holes
 #include <sys/time.h>
 #include <sys/resource.h>
 
-#define DEBUG_CIRCSEGS
+#undef DEBUG_CIRCSEGS
 
 #define ROUND(x) ((long)(((x) >= 0 ? (x) + 0.5  : (x) - 0.5)))
